docs/modules/ROOT/attachments/examples.json

```diff
@@ -54,6 +54,11 @@
     "description": "Demonstrates how to add support for metrics, health checks and distributed tracing",
     "link": "https://github.com/apache/camel-quarkus-examples/tree/main/observability"
   },
+  {
+    "title": "Platform HTTP security",
+    "description": "Demonstrates how to secure platform HTTP with Kyecloak",
```

Comment:
Maybe change the title to `Platform HTTP security with Keycloak`.

Also there's a typo in the description`Kyecloak`.